docs/api/components/schemas.yaml

```diff
@@ -549,9 +549,17 @@ Changes-feed:
             type: array
             items:
               type: object
+              minProperties: 1
+              maxProperties: 1
               properties:
                 rev:
-                  description: The new revision that was caused by that change.
+                  description: |
+                    The new Revision ID associated with the change.
+                    This is usually omitted in favour of the `cv` property when the `version_type` preference is set to `cv`,
+                    however data written prior to Sync Gateway 4.0 can still emit `rev` only change entries if we do not have a `cv` available for that document revision.
+                  type: string
+                cv:
+                  description: The new Current Version associated with the change. This value requires the `version_type` preference set to `cv`.
                   type: string
             uniqueItems: true
```

Comment:
Not sure if a better way to express this is:


```
            items:
              oneOf:
                - type: object
                  properties:
                    rev:
                      description: The new Revision ID associated with the change.
                      type: string
                  title: Revision ID
                - type: object
                  properties:
                    cv:
                      description: The new Current Version associated with the change. This value requires the `version_type` preference set to `cv`.
                      type: string
                  title: Current Version
```                  

Or if this would imply that the all of the items are of the same type.